tests/unittests/test_net.py

```diff
@@ -7759,6 +7771,7 @@ def test_excludes_stolen_macs(self):
                 "aa:aa:aa:aa:aa:02": "enp0s2",
                 "aa:aa:aa:aa:aa:03": "bridge1-nic",
                 "00:00:00:00:00:00": "lo",
+                "aa:aa:aa:aa:aa::04": "netvsc0",
```

Comment:
Perhaps add a test case for the reverse ordering (device list with hv_netvsc first, and another with it second)?

Reply:
Good idea. Added the suggested test. I also added another test for two netvsc nics with same mac would still raise an error.